src/vscode-bicep-ui/apps/deploy-pane/vite.config.ts

```diff
@@ -17,4 +18,21 @@ export default defineConfig({
       },
     },
   },
+  test: {
+    globals: true,
+    watch: false,
+    restoreMocks: true,
+    environment: 'jsdom',
+    coverage: {
+      enabled: true,
+      exclude: ["src/index.tsx", ...coverageConfigDefaults.exclude],
+    },
+    setupFiles: ["src/__tests__/setupGlobals.ts"],
+    deps: {
+      inline: ['@vscode-elements/react-elements'],
+    },
+    alias: {
+      '@vscode-bicep-ui/components': path.resolve(__dirname, '../../packages/components/src/index.ts')
```

Comment:
Feels like there should be a better way to do this, but I couldn't find one.

Reply:
According to https://github.com/vitest-dev/vitest/issues/2922 we need to add an `main` entry in `vscode-bicep-ui/packages/components/package.json`:

![image](https://github.com/user-attachments/assets/92585a40-cc82-4572-b606-cbf81ff6e3c9)

I’ve confirmed that doing so resolves the `Failed to resolve entry...` error.